fdbserver/DataDistribution.actor.cpp

```diff
@@ -941,6 +960,17 @@ struct DDTeamCollection : ReferenceCounted<DDTeamCollection> {
 						if(servers.size() == self->configuration.storageTeamSize || self->satisfiesPolicy(servers, self->configuration.storageTeamSize)) {
 							servers.resize(self->configuration.storageTeamSize);
 							self->addTeam(servers, true);
+//							if ( !self->redundantTeamRemover.isReady() ) {
```

Comment:
Why are these, and similarly some code below, comment'd out?

Reply:
This is a WiP patch set. Those commented codes are for discussion later and may likely be removed in the final version. 

P.S. I will rebase the commits on my branch and update this PR for the real ready-to-merge version.